### ISSUE TITLE
Compatible with defineType/defineField paradigm in studio v3 ?

### ISSUE THREAD

**Issue by droplab** [coreyward/sanity-pills#20]
Love this library, thanks for all the work on it. just wondering if it still makes sense to use now, particularly when it comes to type safey?

**Comment by coreyward** [comment]
Yeah, sanity-pills v2 has been compatible with v3 for a long while now, I just need to do some tidying up so it's clear. Using the `fields` helper even handles the calls to `defineField` for you.
